src/app/register/register.component.html

```diff
@@ -12,9 +12,14 @@
           text="Donor Registration"
         ></biggive-heading>
       </div>
+      @if (redirectPath.startsWith('regular-giving')) {
```

Comment:
Should we standardise these so that everything either does or doesn't include a leading `/`? I think that for redirect purposes it didn't matter, but now that we're checking start of strings specifically it's a bit confusing to introduce variation between Login & Register and makes it harder to see at a glance whether both are going to work.

Reply:
Yes. Done in 0fb53570